testing/run_all_unittests.cc

```diff
@@ -18,5 +19,10 @@ int main(int argc, char** argv) {
 #endif  // OS_IOS
 
   ::testing::InitGoogleTest(&argc, argv);
-  return RUN_ALL_TESTS();
+  auto timeout_listener = new flutter::testing::TestTimeoutListener();
+  auto& listeners = ::testing::UnitTest::GetInstance()->listeners();
+  listeners.Append(timeout_listener);
+  auto result = RUN_ALL_TESTS();
+  delete listeners.Release(timeout_listener);
```

Comment:
Not that it matters much other than for the sake of anti-delete-pedantry but... you could save a line by creating the listener local var with a `make_unique` and then jam the raw pointer into the `listeners` list. 

I suppose the obvious counter-argument is that the `unique` would be an utter lie.